electron/generators/diograph-generator.js

```diff
@@ -0,0 +1,68 @@
+const { isEmpty, promiseAllReduce } = require('../lib/utils')
+const { readPaths } = require('../readers/folder-reader')
+const { generateFileDiory, generateFolderDiory } = require('./diory-generator')
+const { generateFileLink, generateFolderLink } = require('./link-generator')
+
+function getDiorys(diograph, links) {
```

Comment:
I think (and eslint confirms) that functions should be introduced before their usage. I prefer the order they are used. And thus public functions in the bottom :)

If we would use const -notation for function, this would be required by javascript.

Reply:
28:34  warning  'generateDiograph' was used before it was defined  no-use-before-define

This don't have other option as it's recursive.

Reply:
One option is the place private functions inside public but I think it makes readability harder.

Reply:
Okay, interesting... Have to think about this...let's get back to this in next pull requests :)